### PR TITLE
Fix for compile error in queue diff

### PR DESCRIPTION
**Pull request by sven-urbanski-freiheit-com**
This PR fixes a compile error that was accidentally merged.

### CODE REVIEW

File: services/frontend-service/src/ui/ReleaseDialog.tsx [hunk 1]
```diff
@@ -120,15 +125,21 @@ const QueueDiff = (props: { queued: number; current: number; releases: Release[]
...
             </Tooltip>
         );
     }
     return (
         <Tooltip title={prefix + diff + ' behind'}>
             <span>
-                <span className="ahead">{'+' + diff}</span>
+                &nbsp;
+                {' queued: '}
+                <span className="ahead" data-testid="queue-diff">
```
Comment (tameremad) [inline services/frontend-service/src/ui/ReleaseDialog.tsx:140]: Is the className correct here? or should it be `behind`?

File: services/frontend-service/src/ui/ReleaseDialog.tsx [hunk 2]
```diff
@@ -91,26 +91,31 @@ const VersionDiff = (props: { current: number | undefined; target: number; relea
     }
 };
 
-const QueueDiff = (props: { queued: number; current: number; releases: Release[] }) => {
+// target is the version we are looking at currently:
+const QueueDiff = (props: { queued: number; target: number; current: number; releases: Release[] }) => {
```
Comment (tameremad) [inline services/frontend-service/src/ui/ReleaseDialog.tsx:95]: I think we can remove the `current` prop from here?

**Review by tameremad — APPROVED** [review]
:+1:

File: services/frontend-service/src/ui/ReleaseDialog.tsx (hunk 2 above)
Comment (sven-urbanski-freiheit-com) [reply services/frontend-service/src/ui/ReleaseDialog.tsx]: done

File: services/frontend-service/src/ui/ReleaseDialog.tsx (hunk 1 above)
Comment (sven-urbanski-freiheit-com) [reply services/frontend-service/src/ui/ReleaseDialog.tsx]: yes! I totally overlooked the colors.
Done

**Pull request merged**